Src/Support/Google.Apis.Auth/OAuth2/ServiceAccountCredential.cs

```diff
@@ -251,9 +264,14 @@ public override async Task<bool> RequestAccessTokenAsync(CancellationToken taskC
 
         /// <summary>
         /// Gets an access token to authorize a request.
-        /// If <paramref name="authUri"/> is set and this credential has no scopes associated
-        /// with it, a locally signed JWT access token for given <paramref name="authUri"/>
-        /// is returned. Otherwise, an OAuth2 access token obtained from token server will be returned.
+        /// If this credential has <see cref="Scopes"/> associated, but <see cref="UseJwtAccessWithScopes"/> flag
+        /// is not set, an OAuth2 access token obtained from token server will be returned.
+        /// Otherwise, a locally signed JWT access token will be returned. 
+        /// The JWT access token would contain "aud" claim based on <paramref name="authUri"/> if there
+        /// are no <see cref="Scopes"/> associated with the credential or <see cref="UseJwtAccessWithScopes"/> 
+        /// flag is set to false.
+        /// Alternatively, JWT access token would contain "scope" claim based on <see cref="Scopes"/>. 
+        /// The same logic applies when both <paramref name="authUri"/> and <see cref="Scopes"/> are present. 
         /// A cached token is used if possible and the token is only refreshed once it's close to its expiry.
         /// </summary>
         /// <param name="authUri">The URI the returned token will grant access to.</param>
```

Comment:
Maybe add here something like: "Should be specified if no <see cref="Scopes"/> have been specified for the credential."

Reply:
i thought i fixed it, will do, thanks